lib/task.js

```diff
@@ -309,17 +309,24 @@ function factory(
         var self = this;
 
         self.instantiateJob();
+        // Support schedulerOverrides for backwards graph definition compatibility
+        var timeout = self.definition.options.timeout;
```

Comment:
@yyscamper or, as another suggestion along the lines of conventional "magic" values in javascript, what about `$timeout` or `$taskTimeout` ?